manifests/010-prometheus-rules.yaml

```diff
@@ -9,14 +9,48 @@ spec:
   groups:
   - name: SamplesOperator
     rules:
+    - alert: SamplesRetriesMissingOnImagestreamImportFailing
+      expr: sum(openshift_samples_failed_imagestream_import_info) > sum(openshift_samples_retry_imagestream_import_total) - sum(openshift_samples_retry_imagestream_import_total offset 30m)
+      for: 2h
+      labels:
+        severity: warning
+      annotations:
+        message: |
+          Samples operator is detecting problems with imagestream image imports, and the periodic retries of those
+          imports are not occurring.  Contact support.  You can look at the "openshift-samples" ClusterOperator object
+          for details. Most likely there are issues with the external image registry hosting the images that need to
+          be investigated.  The list of ImageStreams that have failing imports are:
+          {{ range query "openshift_samples_failed_imagestream_import_info > 0" }}
+            {{ .Labels.name }}
+          {{ end }}
+          However, the list of ImageStreams for which samples operator is retrying imports is:
+          retrying imports:
```

Comment:
nit:
```suggestion
          However, the list of ImageStreams for which samples operator is retrying imports is:
```

Reply:
Hmm ... the commit suggestion button is greyed out @adambkaplan and when I hover my mouse over the commit suggestion button it says "This suggestion is invalid because no changes were made to the code."

Reply:
suggestion may have come from a prior commit. I think this is good.